easybuild/easyconfigs/n/NFFT/NFFT-3.3.0-foss-2015a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'NFFT'
+version = '3.3.0'
+
+homepage = 'https://www-user.tu-chemnitz.de/~potts/nfft/'
+description = """The NFFT (nonequispaced fast Fourier transform or nonuniform fast Fourier transform) is a C subroutine
+ library for computing the nonequispaced discrete Fourier transform (NDFT) and its generalisations in one or more
+ dimensions, of arbitrary input size, and of complex data."""
```

Comment:
Hi, it looks good overall; 
fyi. I'd capitalize Fast Fourier Transform everywhere, for readability, yet that's up to your liking.